src/pages/ReimbursementAccount/BusinessInfo/substeps/ConfirmationBusiness.tsx

```diff
@@ -124,12 +119,13 @@ function ConfirmationBusiness({reimbursementAccount, reimbursementAccountDraft,
                 />
                 <MenuItemWithTopDescription
                     description={translate('businessInfoStep.incorporationState')}
-                    title={translate(`allStates.${values[businessInfoStepKeys.INCORPORATION_STATE]}.stateName`)}
+                    title={translate(`allStates.${values[businessInfoStepKeys.INCORPORATION_STATE]}.stateName` as TranslationPaths)}
```

Comment:
You can try to get rid of assertion if you can make `values[businessInfoStepKeys.INCORPORATION_STATE]`  be of type `keyof typeof COMMON_CONST.STATES`

Reply:
tried that originally but it doesn't work:
<img width="918" alt="image" src="https://github.com/Expensify/App/assets/32240132/be473185-5923-498b-9c27-1a6724ac2533">

Reply:
I can do an inline assertion for `keyof typeof COMMON_CONST.STATES` but changing the type of `incorporationState` in `ReimbursementAccountDraft` type doesn't work for the `translate` fn